server/src/main/java/com/linecorp/centraldogma/server/storage/encryption/SessionKeyStorage.java

```diff
@@ -247,4 +254,102 @@ void addSessionKeyListener(Consumer<SessionKey> listener) {
             lock.unlock();
         }
     }
+
+    CompletableFuture<Void> rewrapAllSessionMasterKeys() {
+        final List<SessionMasterKey> allSessionMasterKeys = new ArrayList<>();
+        final byte[] prefix = "session/master/".getBytes(StandardCharsets.UTF_8);
+        try (RocksIterator iterator = rocksDbStorage.newIterator(
+                rocksDbStorage.getColumnFamilyHandle(WDEK_COLUMN_FAMILY))) {
+            iterator.seek(prefix);
+            while (iterator.isValid()) {
+                final byte[] key = iterator.key();
+                final String keyStr = new String(key, StandardCharsets.UTF_8);
+
+                if (!keyStr.startsWith("session/master/")) {
+                    break;
+                }
+
+                if ("session/master/current".equals(keyStr)) {
+                    iterator.next();
+                    continue;
+                }
+
+                try {
+                    final SessionMasterKey sessionMasterKey =
+                            Jackson.readValue(iterator.value(), SessionMasterKey.class);
+                    allSessionMasterKeys.add(sessionMasterKey);
+                } catch (JsonParseException | JsonMappingException e) {
+                    throw new EncryptionStorageException(
+                            "Failed to read session master key: " + keyStr, e);
+                }
+
+                iterator.next();
+            }
+        }
+
+        final List<CompletableFuture<SessionMasterKey>> rewrapFutures = new ArrayList<>();
+        for (SessionMasterKey sessionMasterKey : allSessionMasterKeys) {
+            final String oldKekId = sessionMasterKey.kekId();
+            final CompletableFuture<SessionMasterKey> rewrapFuture =
+                    keyWrapper.rewrap(sessionMasterKey.wrappedMasterKey(), oldKekId, kekId)
+                              .handle((newWrappedKey, cause) -> {
+                                  if (cause != null) {
+                                      logger.warn("Failed to rewrap session master key. version: {}",
+                                                  sessionMasterKey.version(), cause);
+                                      return null;
+                                  }
+                                  if (oldKekId.equals(kekId) &&
+                                      sessionMasterKey.wrappedMasterKey().equals(newWrappedKey)) {
+                                      return null;
+                                  }
+
+                                  return new SessionMasterKey(newWrappedKey,
+                                                              sessionMasterKey.version(),
```

Comment:
Question) Similar to the case of `RepositoryEncryptionStorage`, is there no need to increment the version?